source/agora/node/Ledger.d

```diff
@@ -103,6 +109,7 @@ public class Ledger
         NodeConfig node_config, immutable(ConsensusParams) params,
         UTXOSet utxo_set, IBlockStorage storage,
         EnrollmentManager enroll_man, TransactionPool pool,
+        void delegate (const ref Block) @safe onAcceptedBlock = null,
```

Comment:
This should not be an optional parameter (remove `null`).

Reply:
The FullNode always provides a delegate, and a Validator inherits the full node so it has the same behavior. So it is not an optional parameter.

Reply:
That's right. I was concerned about that.